README.md

```diff
@@ -103,6 +103,11 @@ For local file storage, the call will return a URL formed by joining the `downlo
 `download_url_base` query param was included in the storage URI, `get_download_url`
 will raise a `DownloadUrlBaseUndefinedError` exception. (*see* [**file**](#file) *below*)
 
+
+#### `get_sanitized_uri()` ####
+
+Removes the username/password, as well as all query paramters, form the URL.
+
```

Comment:
"parameters"

Reply:
Do we want to add documentation for `remove_user_info()` too, since other users can create and register their own storage providers?